cmake/OpenLocoUtility.cmake

```diff
@@ -194,7 +194,18 @@ function(_loco_add_target TARGET TYPE)
             INTERFACE
                 "${CMAKE_CURRENT_SOURCE_DIR}/include")
     endif()
-    
+    if (_PUBLIC_FILES) 
+        foreach(hdr ${_PUBLIC_FILES})
+            cmake_path(GET hdr STEM hdr_base)
+            cmake_path(GET hdr FILENAME hdr_filename)
+            set(hdr_src "${CMAKE_CURRENT_BINARY_DIR}/${hdr_base}.cpp")
+            file(WRITE ${hdr_src} "#include <OpenLoco/${TARGET}/${hdr_filename}>")
```

Comment:
This wont work if we have multiple levels of include. Will likely need reworked for that will have to use some cmake_path magic.